manifest.yml

```diff
@@ -0,0 +1,16 @@
+applications:
+  - name: ((app-name))
+    buildpacks:
+      - https://github.com/cloudfoundry/nodejs-buildpack
+    routes:
+      - route: ((route-internal))
+      - route: ((route-external))
+    health-check-type: http
+    health-check-http-endpoint: /
+    health-check-invocation-timeout: 60
+    instances: ((instances))
+    disk_quota: 2G
```

Comment:
Does this really need 2G? More curiosity; it isn't a limitation...

Reply:
Probably not anywhere close to that. I'll update in a new PR.